api/src/main/java/io/minio/GetPresignedObjectUrlArgs.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+import io.minio.http.Method;
+
+/** Argument class of MinioClient.getPresignedObjectUrl(). */
+public class GetPresignedObjectUrlArgs extends ObjectArgs {
+  private Method method;
+
+  // default expiration for a presigned URL is 7 days in seconds
+  private static final int DEFAULT_EXPIRY_TIME = 7 * 24 * 3600;
+  // set default expiry as 7 days if not specified.
+  private Integer expires = DEFAULT_EXPIRY_TIME;
```

Comment:
1. Please change `expires` to `expiry` everywhere
2. You could use `int` itself.